src/ServicePulse.Host/vue/src/components/TimeSince.vue

```diff
@@ -1,21 +1,18 @@
-﻿<script setup>
+﻿<script setup lang="ts">
 import { onMounted, onUnmounted, ref } from "vue";
 import moment from "moment";
 
-const props = defineProps({
-  dateUtc: String,
-  default: function () {
```

Comment:
from reading the [doco](https://vuejs.org/guide/components/props.html#prop-validation), I think this was wrong. 
It should have been:
```js
{ 
  dateUtc: { type: String, default: ... },
}
```

Reply:
anyway, I fixed it